### PR TITLE
V2 UI onboard host instruction

### PR DESCRIPTION
**Pull request by milan-deepfence**
Changes proposed in this pull request:
- Onboard: Added instructions for cloud, host and registry

@deepfence/engineering

### CODE REVIEW

File: deepfence_frontend/apps/dashboard/src/features/onboard/components/connectors/clouds/AWSTerraform.tsx [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+import { memo } from 'react';
+import { HiViewGridAdd } from 'react-icons/hi';
+import { useCopyToClipboard } from 'react-use';
+import { Button, Card, Step, Stepper, Typography } from 'ui-components';
+
+import { CopyToClipboardIcon } from '../../../../../components/CopyToClipboardIcon';
+import { usePageNavigation } from '../../../../../utils/usePageNavigation';
+
+export const AWSTerraform = memo(() => {
+  const [_, copyToClipboard] = useCopyToClipboard();
```
Comment (manV) [inline deepfence_frontend/apps/dashboard/src/features/onboard/components/connectors/clouds/AWSTerraform.tsx:11]: add this to copy to clipboard component itself

File: deepfence_frontend/apps/dashboard/src/features/onboard/components/connectors/clouds/AzureConnectorForm.tsx [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+                  'px-4',
+                  'h-fit',
+                  `${Typography.weight.normal} ${Typography.size.xs} `,
+                )}
+              >
+                terraform plan
+              </pre>
+              <CopyToClipboardIcon
+                onClick={() => {
+                  copyToClipboard('$ terraform plan');
```
Comment (manV) [inline deepfence_frontend/apps/dashboard/src/features/onboard/components/connectors/clouds/AzureConnectorForm.tsx:100]: remove $

File: deepfence_frontend/apps/dashboard/src/features/onboard/pages/Connector.tsx [hunk 3]
```diff
@@ -277,9 +277,19 @@ export const AddConnector = () => {
...
-export const Connector = () => {
-  const [tab, setTab] = useState('add');
+export const Connector = ({ page }: { page: string }) => {
+  const [tab, setTab] = useState(page);
+  const { navigate } = usePageNavigation();
 
+  const onTabChange = (tab: string) => {
+    navigate(`/onboard/${tab}`);
+  };
+  console.log(page);
```
Comment (manV) [inline deepfence_frontend/apps/dashboard/src/features/onboard/pages/Connector.tsx:287]: remove

File: deepfence_frontend/apps/dashboard/src/utils/validator.ts [hunk 4]
```diff
@@ -0,0 +1,9 @@
+// not white space on input
+const regex = /^\S*$/;
+
+export const isEmptyString = (value: string) => {
+  if (!value) {
```
Comment (manV) [inline deepfence_frontend/apps/dashboard/src/utils/validator.ts:5]: `return !value.trim().length`

Comment (milan-deepfence) [reply deepfence_frontend/apps/dashboard/src/utils/validator.ts:5]: resolved

File: deepfence_frontend/apps/dashboard/src/features/onboard/pages/Connector.tsx (hunk 3 above)
Comment (milan-deepfence) [reply deepfence_frontend/apps/dashboard/src/features/onboard/pages/Connector.tsx]: resolved

File: deepfence_frontend/apps/dashboard/src/features/onboard/components/connectors/clouds/AWSTerraform.tsx (hunk 1 above)
Comment (milan-deepfence) [reply deepfence_frontend/apps/dashboard/src/features/onboard/components/connectors/clouds/AWSTerraform.tsx:11]: resolved

**Review by manV — APPROVED** [review]

**Pull request merged**